src/components/search/results/variable-view-layout/variable-view-layout.js

```diff
@@ -1,34 +1,33 @@
 import { Fragment } from 'react'
 import { Spin, Typography, Empty } from 'antd'
 import { VariableSearchResults } from './'
+import { VariableSearchResults2 } from './variable-results-new'
```

Comment:
Can we replace this with VariableSearchResults? Basically cleanup so that we don't have two files for variable-results.